### PR TITLE
chore: Improved components list organization

### PR DESCRIPTION
**Pull request by kof**
## Description

1. Rename Text category to Typography
2. Move Content Block to typography
3. move Time to a standalone new "Localization" category
4. move Media section next after Typography

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 0000)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Review by johnsicili — APPROVED** [review]

File: packages/sdk-components-react/src/time.ws.ts [hunk 1]
```diff
@@ -16,14 +16,13 @@ const presetStyle = {
 } satisfies PresetStyle<"time">;
 
 export const meta: WsComponentMeta = {
-  category: "data",
+  category: "localization",
```
Comment (TrySound) [inline packages/sdk-components-react/src/time.ws.ts:19]: Single component category doesn't look good. And it is still fits with "data". Why add now?

**Comment by TrySound** [comment]
List and list item are also typography

**Comment by TrySound** [comment]
The content block could go into general

**Comment by kof** [comment]
> List and list item are also typography

I agree, I guess I was influenced by Webflow's choice there ... but thinking about it lists is basically part of content authoring ...

**Comment by kof** [comment]
> Then content block could go into general

Hmm difficult choice, I was trying to put all the content authoring things into one place but yeah Content Block is more than Typography.

What if we called it "Content" instead of Typography and kept things like we have them there now + lists?

**Comment by kof** [comment]
The problem with "Content" is that everything is content, including media category

**Comment by kof** [comment]
or alternatively keeping it at Typography and Content Block there as well. Thinking of: content block is more than text, but in the end this component is enabling user to edit text in more complex layouts, but in the end the goal is text editing, which is closer to typography than to anything else ... I think it should stay in Typography

File: packages/sdk-components-react/src/time.ws.ts (hunk 1 above)
Comment (kof) [reply packages/sdk-components-react/src/time.ws.ts:19]: The reason anybody would use Time component is localizing date/time format. They don't need it if they want to just type some date and that has also very little to do with the fact data is remote. Remote nature of data in this case is coinscidence.

IMO user will think: "How do I localize the date" when trying to find that component in the panel

Comment (kof) [reply packages/sdk-components-react/src/time.ws.ts:19]: I agree single component doesn't look good, lets add currency localization, this is very needed soon for shopify

**Comment by kof** [comment]
Regarding list and list item, the question is if having it in typography isn't confusing for someone who wants to use a list to build a menu of some kind

**Comment by TrySound** [comment]
> Regarding list and list item, the question is if having it in typography isn't confusing for someone who wants to use a list to build a menu of some kind

There was never a clear reason why people use lists for menus. They actually harm a11y. So it better be confusing.

**Comment by kof** [comment]
> > Regarding list and list item, the question is if having it in typography isn't confusing for someone who wants to use a list to build a menu of some kind
> 
> There was never a clear reason why people use lists for menus. They actually harm a11y. So it better be confusing.

I guess historically this was the way to go, but we can move past that now

**Pull request merged**